src/operations.c

```diff
@@ -26,11 +26,12 @@
 #include "operations.h"
 
 static int iterate_json(FILE *input, rodsEnv *env, baton_json_op fn,
-                        operation_args_t *args, int *item_count) {
+                        operation_args_t *args,
+                        int *item_count, int *error_count) {
     time_t connect_time = 0;
     int       reconnect = 0; // Set to 1 when reconnecting
     rcComm_t *conn      = NULL;
-    int error_count     = 0;
```

Comment:
We should probably still zeroise error_count here?

Reply:
It's a pointer passed in by the caller, so the caller owns it. Initializing the value  is the caller's job - it was done on line 133.